app/main/posts/detail/post-messages.directive.js

```diff
@@ -51,10 +52,14 @@ function (
                 // Initialize
                 if ($scope.post.contact.contact) {
```

Comment:
@willdoran maybe `$scope.post.contact && $scope.post.contact.contact ` would be safer in the off chance that post.contact  is not there ? Not sure if that's even possible (not being there), just asking

Reply:
ah ok nvm, just realized you are using the contact.id in the else so I'm assuming it's going to be available. Sorry

Reply:
done